internal/label/pullrequests.go

```diff
@@ -12,29 +12,26 @@ import (
 )
 
 // PullRequests applies internal or community labels to pull requests
-func PullRequests(githubClient *github.Client, internalTeam string, cfg config.LabelConfig) error {
+func PullRequests(githubClient *github.Client, internalTeam string, cfg *config.Config) error {
 	teamMembers, err := github2.GetTeamMemberList(githubClient, internalTeam)
 	if err != nil {
 		return fmt.Errorf("error getting team members: %w", err) // Properly handle and return error if team member list fetch fails
 	}
 
-	pullRequests, err := github2.FindCommunityPRs(cfg.LabelCheckRepos, teamMembers, githubClient)
+	pullRequests, err := github2.FindCommunityPRs(cfg, teamMembers, githubClient)
 	if err != nil {
 		return fmt.Errorf("error finding community PRs: %w", err) // Handle error from finding community PRs
 	}
 
 	for _, pullRequest := range pullRequests {
 		user := *pullRequest.User.Login
-		if cfg.LabelSkipMap[user] {
-			continue
-		}
+
```

Comment:
Does this function no longer skip users in the user skipping map?

Reply:
Ah it looks like this actually gets done upstream in the FindCommunityPRs function now. So yeah it's still skipping them, just in a different spot.